sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -43,13 +55,10 @@ def iterable():
 def error():
     return Response(stream_json_error(), status=400)
 
-@streams_api.route('/string', methods=['GET'])
-def string():
-    return Response(
-        streaming_test(), status=200, mimetype="text/plain"
-    )
-    
```

Comment:
I think L46-L51 might have been accidentally removed - the tests are failing because a few of them depend on this code.